writeups/owasp_in_sdlc/index.md

```diff
@@ -3,6 +3,7 @@ title: OWASP in SDLC
 layout: col-document
 tags: SDLC standards integration
 document: integration standards
+redirection_from: "/writeups/owasp_in_sdlc/owasp_in_sdlc.html"
```

Comment:
Do you know that it's linked elsewhere as *.html?
You may also want to redirect from: https://owasp.org/www-project-integration-standards/writeups/owasp_in_sdlc/owasp_in_sdlc

(People shouldn't be using filename extensions.)